lib/krane/kubernetes_resource.rb

```diff
@@ -442,8 +473,6 @@ def global?
       @global || self.class::GLOBAL
     end
 
-    private
```

Comment:
why was this deleted?

Reply:
Sorry, I was trying something to replace `self.extract_count_timestamp` and forgot to put it back - fixed.

Reply:
Sorry, I was trying something to replace self.extract_count_timestamp and forgot to put it back - fixed.